_posts/2023-12-13-release-0_4_0.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "kroxylicious release 0.4.0"
+date:   2023-12-14 00:00:00 +1300
+author: "Keith Wall"
+author_url: "https://www.github.com/k-wall"
+categories:  releases kroxylcious topic-encryption
+---
+
+The Kroxylicious project is very pleased to announce the release of Kroxylicious 0.4.0.   The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for the Apache Kafka(tm) which is _transparent_ to both clients and brokers.
```

Comment:
I'm in two minds about this, its true, but I worry it makes it sound like its finished. Saying its the first iteration isn't very helpful either as we don't actually expect the the filter to change most of the remaining work is around the non functional requirements or reliability. 

Maybe adding another sentence along the lines of 
> We have more work planned to iterate on the implentation to reach production quality. Checkout our [0.5.0](www.github.com/kroxylicious/...) milestone for details

Reply:
I agree it is important that we get over this isn't finished and folks shouldn't be dropping into production.  I think your words convey that well.  I'll add them.